runOrthopairs.sh

```diff
@@ -4,6 +4,27 @@ set -e
 DIR=$(dirname "$(readlink -f "$0")") # Directory of the script -- allows the script to invoked from anywhere
 cd $DIR
 
+## Download files
+echo "Downloading ftp://ftp.pantherdb.org/ortholog/current_release/Orthologs_HCOP.tar.gz..."
```

Comment:
I'm confused... why are these downloads in a script, and other downloads are executed by Jenkins (via Jenkinsfile)?

Reply:
The bash script is meant to exist as an easy way of running the Orthopairs stuff in a pinch, so if you are doing testing or if Jenkins isn't cooperating for whatever reason. I haven't done this yet for the other steps cause its lower priority, but Orthopairs already had a bash script runner so I wanted to keep it up to date

Reply:
I'm also wondering about this - is this shell script an independent way to run outside of Jenkins?

Reply:
@cookersjs Ok, that wasn't clear by looking at the PR, but with this explanation, it sounds fine.

Reply:
Yes it's an independent way to run it outside of Jenkins; Sorry for not making it clearer -- the `runOrthopairs.sh` script has been there for a while, this is just the first time its been updated since it was first created